app/webpacker/css/admin/components/navigation.scss

```diff
@@ -77,8 +77,7 @@ nav.menu {
       color: $color-1 !important;
       text-transform: uppercase;
       position: relative;
-      text-align: center;
-      font-weight: 600;      
+      text-align: center;   
```

Comment:
Except you accidentally left the indent spaces ;)
```suggestion
      text-align: center;
```

Reply:
Thanks ;)